package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@siggame/colisee-builder",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Minor issue, this doesn't need to be changed if you dont want.

You don't have to increment the version if you're just developing on a branch. Only when it touches master/stable.

Or in the case of our modules, when its published